chain/client/src/client.rs

```diff
@@ -807,6 +810,166 @@ impl Client {
         }
     }
 
+    /// Processes received block. Ban peer if the block header is invalid or the block is ill-formed.
+    // This function is just a wrapper for process_block_impl that makes error propagation easier.
+    pub fn receive_block(
+        &mut self,
+        block: Block,
+        peer_id: PeerId,
+        was_requested: bool,
+        apply_chunks_done_callback: DoneApplyChunkCallback,
+    ) {
+        let hash = *block.hash();
+        let prev_hash = *block.header().prev_hash();
+        let _span = tracing::debug_span!(
+            target: "client",
+            "process_block",
+            me = ?self.validator_signer.as_ref().map(|vs| vs.validator_id()),
+            prev_hash = %prev_hash,
+            %hash,
+            height = block.header().height(),
+            %peer_id,
+            was_requested)
+        .entered();
+
+        match self.receive_block_impl(
+            block,
+            peer_id.clone(),
+            was_requested,
+            apply_chunks_done_callback,
+        ) {
+            Ok(_) => {}
+            // Log the errors here. Note that the real error handling logic is already
+            // done within process_block_impl, this is just for logging.
```

Comment:
Let's but the comment on top of the whole match? Or maybe rewrite it as 

```
let res = self.receive_block_impl();
if let Err(err) = res {
            // Log the errors here. Note that the real error handling logic is already
            // done within process_block_impl, this is just for logging.
}
```

THe problem I am trying to solve here is that it's difficult at a glance to tell that we really just logging the errors here, without any kind of processing.